news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "Release note Release Candidate 1.7.6"
+subtitle: "A few weeks after the beta, here it is: the first release candidate for PrestaShop 1.7.6.0 is now ready for you to test!"
+date:   2019-06-25 16:00:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - rc
+ - minor
+ - releases
+ - 1.7.6.x
+---
+
+
+This is the last step before releasing the final 1.7.6.0. It aims at making sure that all regressions found during the beta have been fixed, and that no new ones appeared. If no new regression is reported by June 27, we will make the final version of 1.7.6.0 available.
+
+Please note that it is not currently possible to perform an automatic upgrade from the Beta to the Release Candidate. Furthermore, it will not be possible to upgrade from this Release Candidate to the final release. If you perform an update of your test store to the RC version, you will need to rollback to your current version before being able to upgrade to the final 1.7.6.0.
+
+Go download and install this RC version as a test store, either on your machine or your web hosting, then play with it – and give us your feedback! We need YOU to test this new version of PrestaShop.
+
+**Remember, this release candidate version is pre release software. Do not use it on your production shop!**
+
+
+## New in the release candidate
+
+**Back Office:**
+  - [Introduced an easy way for hooked modules on entity pages to handle error display](https://github.com/PrestaShop/PrestaShop/pull/14239) (a demo module is now finished and available [here](https://github.com/friends-of-prestashop/demo-cqrs-hooks-usage-module))
+
+**Front Office:**
+  - [Improved error messages displayed when spam protection rejects a customer subscription](https://github.com/PrestaShop/PrestaShop/pull/14138) and you can now translate them specifically for a better user experience
+
+**Core:**
+  - [Module Faceted search version has been bumped to 3.0.5](https://github.com/PrestaShop/PrestaShop/pull/14284) which contains interesting bug fixes
+  - Watermark and Product comments modules have been integrated and can now be installed from the Module Catalog page in the back office
+  - Tips and Updates module (psaddonsconnect) has been updated to the 2.0.0 version
+  - [We have replaced PHPExcel by PHPSpreadsheet](https://github.com/PrestaShop/PrestaShop/pull/14316) in order to better handle XLS or XLSX files import
+  
+  
+## Changelog
+
+100+ pull requests have been merged since the beta release and 50+ regressions have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. A huge thanks to everyone involved in testing and fixing issues during the beta phase, and especially 202 ecommerce, OneDotIT, MartinF who helped us a lot during this period !
```

Comment:
and especially 202 ecommerce, OneDotIT and MartinF who helped us a lot during this period !

Reply:
done